src/containers/edit-profile/notification-tab/NotificationTab.constants.tsx

```diff
@@ -0,0 +1,22 @@
+export const titleWithSubtitle = [
+  {
+    id: '1',
+    title: 'editProfilePage.profile.notificationsTab.offerStatus',
+    subtitle: 'editProfilePage.profile.notificationsTab.offerSubtitle'
+  },
+  {
+    id: '2',
+    title: 'editProfilePage.profile.notificationsTab.chat',
+    subtitle: 'editProfilePage.profile.notificationsTab.chatSubtitle'
+  },
+  {
+    id: '3',
+    title: 'editProfilePage.profile.notificationsTab.similarOffers',
+    subtitle: 'editProfilePage.profile.notificationsTab.similarOffersSubtitle'
+  },
+  {
+    id: '4',
```

Comment:
we are using id just for `key` value, as Bohdan mentioned in one of my PRs it's redundant, we can just use title for a key
[You can check it here](https://github.com/ita-social-projects/SpaceToStudy-Client/pull/1840#discussion_r1582860650)

Reply:
Thanks a lot, fixed